src/VscPreviewExtension/src/contentProvider.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { workspace, ExtensionContext, TextDocumentContentProvider, EventEmitter, Event, Uri } from "vscode";
+import * as path from "path";
+
+export class ContentProvider implements TextDocumentContentProvider {
+    private _context: ExtensionContext;
+    private _onDidChange = new EventEmitter<Uri>();
+    protected _content: string;
+    public port;
+
+    constructor(context: ExtensionContext) {
+        this._context = context;
+    }
+
+    protected getMediaPath(mediaFile): string {
```

Comment:
it will be parsed  when the function is called,but adding type here maybe more readable

---
In reply to: [80382318](https://github.com/dotnet/docfx/pull/716#discussion_r80382318) [](ancestors = 80382318)